conf/igenomes.config

```diff
@@ -9,139 +9,412 @@
 
 params {
   // illumina iGenomes reference file paths
-  // TODO nf-core: Add new reference types and strip out those that are not needed
   genomes {
     'GRCh37' {
-      bed12   = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Annotation/Genes/genes.bed"
-      fasta   = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Sequence/WholeGenomeFasta/genome.fa"
-      gtf     = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Annotation/Genes/genes.gtf"
-      star    = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Sequence/STARIndex/"
+      fasta       = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Sequence/WholeGenomeFasta/genome.fa"
+      bwa         = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Sequence/BWAIndex/genome.fa"
+      bowtie2     = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Sequence/Bowtie2Index/"
+      star        = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Sequence/STARIndex/"
+      bismark     = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Sequence/BismarkIndex/"
+      gtf         = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Annotation/Genes/genes.gtf"
+      bed12       = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Annotation/Genes/genes.bed"
+      readme      = "${params.igenomes_base}/Homo_sapiens/Ensembl/GRCh37/Annotation/README.txt"
+      mito_name   = "MT"
+      macs_gsize  = "2.7e9"
+      blacklist   = "${baseDir}/assets/blacklists/GRCh37-blacklist.bed"
```

Comment:
These blacklist files does remind me of something.
@nf-core/core Anyone remember where we talked about that?
@drpatelh maybe?
Shouldn't it be a resource to upload to AWS iGenomes actually?

Reply:
ATACseq calling!

Reply:
@drpatelh for sure ;-)

Reply:
Yep. They were supposed to be put on AWS iGenomes a couple of years ago but hasn't happened :smirk: I added this all to the template too I think so shouldn't be a problem if they aren't being used. If they are then just copy and paste them there :sweat_smile:

Reply:
PS: Sorry @t-neumann ! I did see your review request but I have been quite preoccupied with the `viralrecon` pipeline of late.

Reply:
Other people have managed to get stuff added to iGenomes - you just need to prep everything for me (ask @MaxUlysse 😉 )